README.md

```diff
@@ -318,7 +318,7 @@ To return additional custom headers in the response, set the headers map in the
 In this example the headers x-requested-by and dummy will be returned on the response.  contentType can be specified separately, as it is above, or specified as "content-type" in the "headers" map.
 
 ### Templating your JSON
-You can take values in the route and insert them into your json. All you need to do is set "enableTemplate" to true, specify a content type and have a matching @ in the mock json file. Here's an example:
+You can take values in the route and insert them into your json. All you need to do is set "enableTemplate" to true, specify a content type and have a matching @ (or in the case of unquoted numbers, use @@) in the mock json file. Here's an example:
```

Comment:
You can take values in the route and insert them into the response json. To enable this, you identify a placeholder in your request by inserting a colon followed by an identifier where you would like the value extracted from.  You then set "enableTemplate" to true, specify a content type, and in your response file, wherever you want that value to appear, you insert the '@' symbol followed by your identifier.  This placeholder can appear anywhere in your json template, including in multiple places.

There's also a special case where you may want the value for a json key to be returned as a numeric value, not a value wrapped in quotes. However, json doesn't allow you to specify a '@' outside of quotes in the template.  In that case, you can use a special version of the prefix, '@#' which tells the template parser to also remove the quotes immediately before and after the placeholder as part of the templating process.

Reply:
took guidance and copy from your description and incorporated it into the updated README.md
kept the prefix as @@ however per your original suggestion.